site/content/docs/5.0/forms/checks-radios.md

```diff
@@ -211,6 +211,47 @@ Omit the wrapping `.form-check` for checkboxes and radios that have no label tex
 </div>
 {{< /example >}}
 
+## Centering
+
+Omit the wrapping `.form-check` and `.form-check-inline` and use [spacing utilities]({{< docsref "/utilities/spacing" >}}) for the label to center the checkbox or radio.
```

Comment:
This does not truly reflect your examples since centering is due to `.text-center`, not spacing utilities.
There're several ways to accomplish this (e.g. `d-flex justify-content-center`), I think it worth mentioning it.

Reply:
Absolutely fair enough - made sense in my head (the bane of all editing...) but wasn't conveyed accurately. Updated text and examples to clarify and show alternative centering methods.